src/mvs_eland/C0_data_processing.py

```diff
@@ -1710,8 +1711,13 @@ def add_maximum_cap(dict_values, group, asset, subasset=None):
     else:
         dict = dict_values[group][asset][subasset]
     if MAXIMUM_CAP in dict:
-        # check if maximumCap is greater that installedCap
         if dict[MAXIMUM_CAP][VALUE] is not None:
+            # adapt maximumCap of nocolan-dispatchable sources
+            if group == ENERGY_PRODUCTION and dict[RENEWABLE_ASSET_BOOL][VALUE] == True:
```

Comment:
> The decisive parameter right now would rather be `if group == ENERGY_PRODUCTION and dict[FILENAME]!=None and dict[TIMESERIES_PEAK]!=1`. (evtl. kannst du alternativ auch `if TIMESERIES in dict` testen, aber ich bin nicht sicher).
> 

I'm not sure about that as I'm looking for max(timerseries_normalized) =! 1 - isn't TIMESERIES_PEAK the peak of TIMESERIES? And even if the peak of the normalized time series is 1 it wouldn't change the time series dividing by 1.

I'll use `dict[FILENAME]!=None` instead of `dict[RENEWABLE_ASSET_BOOL][VALUE] == True` as you suggest.

> Btw... is it allowed to define `dict`, as it is also a type? I find it confucting. Can you change this to `dict_asset`?

Yes it's allowed and I left it like that as it was defined like that already in this function. I change it to `asset_dict`  being the dict of the (sub)-asset.